### PR TITLE
Clarify ingest tool/library names for Ingest CLI and Python Ingest library

### PR DESCRIPTION
**Pull request by Paul-Cornell**
For API-initiated ingest workflows, there has been a conflation and confusion of "CLI," "Python SDK," and "Python Ingest" for tool/library names throughout the docs. This PR seeks to separate these names and to help reduce confusion here. 

Specifically, customers seem to be confused--and the docs have seemed to reinforce this confusion--with using tool/library names for ingestion such as:

- "CLI" - The CLI can only handle ingest operations in Unstructured. So, preface it with "Ingest" to make that clearer.
- "Python SDK" - The Python SDK cannot do multi-file batch ingestion operations by itself. So, distinguish the SDK from the "Ingest Python library."
- "Open source" - There is an imminent change with the "Ingest Python library" repo being separated from the open source library repo. So, proactively begin distinguishing the "open source library" from the "Ingest Python library."

Note that there is an open discussion about whether to call it the "Ingest Python SDK" or the "Ingest Python API." So as to not hold up this PR, I am using the more general term "Ingest Python library" in this PR for now. I can certainly create a follow-on PR to replace "library" with "SDK" or "API" once a consensus decision is reached here.

### CODE REVIEW

File: api-reference/ingest/destination-connector/astra.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import SharedAPIKeyURL from '/snippets/general-shared-text/api-key-url.mdx';
 <SharedAstra />
 <SharedAPIKeyURL/>
 
-Now call the Unstructured CLI or Python SDK. The source connector can be any of the ones supported. This example uses the local source connector:
+Now call the Unstructured Ingest CLI or Unstructured Ingest Python. The source connector can be any of the ones supported. This example uses the local source connector:
```
Comment (scanny) [inline api-reference/ingest/destination-connector/astra.mdx:11]: `Unstructured Ingest Python` seems an odd name. What would that mean?

I could possibly see `Unstructured Ingest Python API`. Is that what we're talking about?

File: faq/faq.mdx [hunk 2]
```diff
@@ -44,7 +44,7 @@ Yes, you can still use your old API keys. We will migrate all the user keys to t
 
 When you log in to the Serverless API dashboard, you can access your API keys by clicking the `API Keys` link in the side navigation.
 Under the `Actions` column, click the `Copy` icon to copy the key or the boilerplate codes to process the documents
-using the Unstructured REST API POST with `curl`, or the Unstructured CLI, or the [Unstructured Python SDK](https://github.com/Unstructured-IO/unstructured-python-client) or [Unstructured JavaScript/Typescript SDK](https://github.com/Unstructured-IO/unstructured-js-client).
+using the Unstructured REST API POST with `curl`, or the [Unstructured Ingest CLI](/ingestion/overview#unstructured-ingest-cli), or the [Unstructured Python SDK](https://github.com/Unstructured-IO/unstructured-python-client) or [Unstructured JavaScript/Typescript SDK](https://github.com/Unstructured-IO/unstructured-js-client).
```
Comment (scanny) [inline faq/faq.mdx:47]: "Unstructured REST API POST" seems an odd name, not sure what that would refer to. I'd be inclined to just say "Unstructured REST API". I don't think the detail that a `curl` request to the API uses the `POST` HTTP method is necessary here. Better to link to an example that includes that detail.

Btw, "boilerplate codes" just above strikes me as misspoken; "boilerplate code" would get it done I expect.

**Review by scanny — CHANGES_REQUESTED** [review]
We're going to need a different name instead of "Unstructured Ingest Python". Happy to noodle that with you if no clear alternative presents itself.

File: api-reference/ingest/destination-connector/astra.mdx (hunk 1 above)
Comment (Paul-Cornell) [reply api-reference/ingest/destination-connector/astra.mdx:11]: There is an open discussion about whether to call it the "Unstructured Ingest Python SDK" or the "Unstructured Ingest Python API." So as to not hold up this PR, I am using the more general term "Unstructured Ingest Python library" in this PR for now. I can certainly create a follow-on PR to replace "library" with "SDK" or "API" once a consensus decision is reached here.

File: faq/faq.mdx (hunk 2 above)
Comment (Paul-Cornell) [reply faq/faq.mdx:47]: Sounds good: removed `POST` and `curl`; replaced "boilerplate" with "example."

**Review by MKhalusova — APPROVED** [review]
I think this is a reasonable naming choice.

**Pull request merged**